packages/grafana-ui/src/components/Text/Text.tsx

```diff
@@ -6,8 +6,8 @@ import { GrafanaTheme2, ThemeTypographyVariantTypes } from '@grafana/data';
 import { useStyles2 } from '../../themes';
 
 export interface TextProps {
-  /** Defines what HTML element is defined underneath */
-  as: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'span' | 'p' | 'legend';
+  /** Defines what HTML element is defined underneath. "span" by default */
+  element?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'span' | 'p' | 'legend';
```

Comment:
I do like this change but I'm not sure it is worth introducing a breaking change just for the sake of changing the name of the property.

Is the Text component widely used by plugins? Have you done any research on this?

Reply:
I saw (further down in the PR) that we expose this one via `unstable.ts`. I guess this is a new pattern we have introduced. @jackw something you have been involved in?

Reply:
Hi @mckn,
The `unstable` package has been available only to grafana core.
We create this package to test the component and give the opportunity to use it by 'core' devs, so they can give us feedback regarding some concerns we had while we were building it.
With these latest changes, we are about to move the `Text` component to `grafana-ui`.

Reply:
Considerable conversation here. https://raintank-corp.slack.com/archives/CT7S455SP/p1684324377579169

tl;dr - this component is not exposed publicly and was never stable a stable API. The point of this process is to make these "breaking changes" without actually impacting plugins.

Reply:
Ah, ok - thanks for the clarification 🙏🏻